packages/stream-metadata/src/environment.ts

```diff
@@ -26,12 +27,17 @@ function makeConfig() {
 	// eslint-disable-next-line no-process-env -- this is the only line where we're allowed to use process.env
 	const env = envSchema.parse(process.env)
 	const web3Config = getWeb3Deployment(env.RIVER_ENV)
+	const baseUrl = new URL(env.RIVER_STREAM_METADATA_BASE_URL)
+	if (env.RIVER_ENV.startsWith('local')) {
+		baseUrl.port = env.PORT.toString()
+	}
 
 	return {
 		web3Config,
 		riverEnv: env.RIVER_ENV,
 		baseChainRpcUrl: env.BASE_CHAIN_RPC_URL,
 		riverChainRpcUrl: env.RIVER_CHAIN_RPC_URL,
+		riverSpaceStreamBaseUrl: `${baseUrl.origin}/space`.toLowerCase(),
```

Comment:
same comments apply here. dev environments should set the proper BASE_URL and call it

Reply:
talked offline